charts/kube-shutdown-after/templates/deployment.yaml

```diff
@@ -29,18 +29,6 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          ports:
```

Comment:
intended?

Reply:
oh, just saw the comments... yeah, pretty sure they aren't needed.

Reply:
Yeap, that's it.